features/learn2earn/api/build.gradle.kts

```
@@ -0,0 +1,5 @@
+plugins {
+    alias(deps.plugins.android.library)
+    alias(deps.plugins.kotlin.android)
+    id("configuration")
+}
```

Тут точно нужен android, кажется тут можно без него обойтись?
```suggestion
plugins {
    alias(deps.plugins.kotlin.jvm)
    id("configuration")
}
```